jakelib/scripts.jake

```diff
@@ -0,0 +1,58 @@
+const { cyan, green, magenta, red } = require('chalk');
+
+const { question } = require('readline-sync');
+
+const CLUSTERS = {
```

Comment:
We should import this from a shared place, so that we don't have to duplicated it from pipeline.